modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPackage.java

```diff
@@ -1161,11 +1161,11 @@ public void create(TileEntry entry) throws IOException {
                 }
                 //create the tile table itself
                 st = cx.prepareStatement(format("CREATE TABLE %s (" +
-                    "id INTEGER PRIMARY KEY AUTOINCREMENT," +
-                    "zoom_level INTEGER NOT NULL DEFAULT 0," +
-                    "tile_column INTEGER NOT NULL DEFAULT 0," +
-                    "tile_row INTEGER NOT NULL DEFAULT 0," +
-                    "tile_data BLOB NOT NULL DEFAULT (zeroblob(4)))", e.getTableName()));
+                    "id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT," +
```

Comment:
This change is arguable. Table 11 in the released GeoPackage 1.1 spec and Table 10 in the GeoPackage 1.2 spec say not null, but the CITE suite says null. Will see what the results of a change to the ETS show.